indico/modules/rb/schemas.py

```diff
@@ -386,6 +386,7 @@ class Meta:
     link_id = fields.Int()
     link_back = fields.Bool(load_default=False)
     admin_override_enabled = fields.Bool(load_default=False)
+    extra_fields = fields.Dict(load_default={})
```

Comment:
```suggestion
    extra_fields = fields.Dict(load_default=lambda: {})
```

Mutable defaults are dangerous

Reply:
Ugh, good catch. Thanks 👍